CPS-0018/README.md

```diff
@@ -0,0 +1,136 @@
+---
+CPS: 18
+Title: Greater Transaction Throughput
+Category: Consensus
+Status: Open
+Authors:
+  - Arnaud Bailly <arnaud.bailly@iohk.io>
+  - Brian W. Bush <brian.bush@iohk.io>
+  - Hans Lahe <hans.lahe@iohk.io>
+Implementors: Intersect
+Discussions: 
```

Comment:
```suggestion
Discussions: 
  - https://github.com/cardano-foundation/CIPs/pull/925/
```
worth putting this PR under discussions

Reply:
yes @bwbush this is definitely useful to mark the introduction of the CPS, even though technical discussion hasn't occured here (yet).